web/src/sidebar_ui.js

```diff
@@ -181,4 +182,43 @@ export function initialize_right_sidebar() {
             }
         }
     });
+
+    $("#buddy-list-users-matching-view-container").on(
+        "click",
+        ".buddy-list-subsection-header",
+        () => {
+            $("#buddy-list-users-matching-view").toggleClass("collapsed");
+            const is_collapsed = $("#buddy-list-users-matching-view").hasClass("collapsed");
+            $("#buddy-list-users-matching-view-container .toggle-narrow-users").toggleClass(
+                "fa-caret-down",
+                !is_collapsed,
+            );
+            $("#buddy-list-users-matching-view-container .toggle-narrow-users").toggleClass(
+                "fa-caret-left",
+                is_collapsed,
+            );
+        },
+    );
+
+    $("#buddy-list-other-users-container").on("click", ".buddy-list-subsection-header", () => {
+        $("#buddy-list-other-users").toggleClass("collapsed");
+        const is_collapsed = $("#buddy-list-other-users").hasClass("collapsed");
+        $("#buddy-list-other-users-container .toggle-other-users").toggleClass(
+            "fa-caret-down",
+            !is_collapsed,
+        );
+        $("#buddy-list-other-users-container .toggle-other-users").toggleClass(
+            "fa-caret-left",
+            is_collapsed,
+        );
+    });
+
+    // There's no way to get to the subscriber view by url,
+    // so we have this click handler to change the tab.
+    // Ideally in the future we have a url structure to go
+    // directly to the subscriber settings without this.
```

Comment:
Discussion about that on CZO [here](https://chat.zulip.org/#narrow/stream/6-frontend/topic/open.20stream.20subscribers/near/1676211)